src/docfx/SubCommands/PdfCommand.cs

```diff
@@ -194,6 +194,11 @@ private static void MergeOptionsToConfig(PdfCommandOptions options, PdfJsonConfi
             {
                 config.NoInputStreamArgs = options.NoInputStreamArgs.Value;
             }
+
+            if (!string.IsNullOrEmpty(options.FilePath))
+            {
+                config.Wkhtmltopdf.FilePath = options.FilePath;
```

Comment:
If it is a relative path from command line, users should expect it based on current directory instead of [config path](https://github.com/dotnet/docfx/pull/5916/files#diff-5a8e6fb1e4e267627724a0cc6afc5a69R30). We can apply a simila logic here like `config.Wkhtmltopdf.FilePath = options.FilePath?.GetFullFilePath(Environment.CurrentDirectory)`.